crates/runtime/src/dataconnector/github.rs

```diff
@@ -209,7 +213,80 @@ impl GithubTableArgs for IssueTableArgs {
             name = self.repo
         );
 
-        (query.into(), "/data/repository/issues/nodes".into(), 1)
+        let gql_schema = Arc::new(Schema::new(vec![
```

Comment:
this schema looks static, and is quite long? we should move it into a separate function, like `IssueTableArgs::schema()`

Reply:
Yeah 👍, plan to improve this when adding schema for the next GH connector table